tools/build_config/build.ps1

```diff
@@ -216,3 +271,11 @@ if ($test -eq $true) {
 if ($package -eq $true) {
   Invoke-Package -build_dir "$build_dir"
 }
+
+if ($docs -eq $true) {
+  Invoke-Docs
+}
+
+if ($push_docs -eq $true) {
+  Invoke-Push
+}
```

Comment:
I realise you were just following what was done previously, but these `-eq $true`s can be removed from the `if` statements

Reply:
Does that mean you want all of them removing?